src/client/setup.ts

```diff
@@ -46,7 +48,13 @@ function twitter(): void {
 function performanceMetrics(): void {
     window.addEventListener(
         'load',
-        () => console.log('Metrics: ', metrics(performance.getEntries())),
+        () => {
+            pipe2(
```

Comment:
guess pipe is quite small so doesn't add much to the client side bundle?

Reply:
Yup, I suspect this will pull in the `lib` module because I don't *think* we're tree shaking atm.